slider.html

```diff
@@ -11,24 +11,23 @@
 

```

Comment:
now you have only 3 images. what are you going to do in the case when you have 100 images? you should not write a lot of `myslides` div. your js code should return divs with content.